lib/SelectionBar/SelectionBarLeft.js

```diff
@@ -0,0 +1,28 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import pluralize from 'pluralize';
+import { Button } from '../';
+
+const SelectionBarLeft = props => (
```

Comment:
`SelectionBarLeft` applies that there is some logic or at least style hooks for the left position. It seems though that isn't the case. Should the name be more directed at the contents of the component?